### DISCUSSION TITLE
Callout in Nextra 2.10.0

### DISCUSSION THREAD

**Discussion by eat-sleep-code** [shuding/nextra#2138] [General]
How do you implement the Component in Nextra 2.10.0?

It doesn't appear to be documented in the latest docs.   Has it been removed?

https://nextra.site/docs/docs-theme/built-ins

**Comment by dimaMachina** [comment]
https://the-guild.dev/blog/nextra-3#steps-callout-tabs-cards-and-filetree-exports-were-removed
